src/test/scala/uk/gov/hmrc/play/filters/CacheControlFilterSpec.scala

```diff
@@ -109,11 +111,8 @@ class CacheControlFilterSpec extends WordSpecLike with Matchers with MockitoSuga
   }
 
   "Creating the filter from config" should {
-    "load the correct values" in new WithApplication(FakeApplication(additionalConfiguration = Map("caching" -> List("image/", "text/")))) {
-      CacheControlFilter.fromConfig("caching").cachableContentTypes should be (List("image/", "text/"))
-    }
-    "throw an exception on missing config" in  {
-      an [RuntimeException] should be thrownBy CacheControlFilter.fromConfig("caching").cachableContentTypes
```

Comment:
What's the justification for dropping the test for "throw an exception on missing config"?